include/constants/pokemon.h

```diff
@@ -230,6 +230,9 @@
 #define SPLIT_SPECIAL   0x1
 #define SPLIT_STATUS    0x2
 
+// Just for readability
+#define ALWAYS_CRIT         3
```

Comment:
This is a misnomer, since Gens 2-5 doesn't guaranteee a critical hit if the ratio is 3.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/9f8cba54-071b-45ba-a02f-14facb64c4b5)